pandas/tests/resample/test_base.py

```diff
@@ -93,13 +93,13 @@ def test_raises_on_non_datetimelike_index():
 
 @all_ts
 @pytest.mark.parametrize("freq", ["M", "D", "H"])
-def test_resample_empty_series(freq, empty_series, resample_method):
+def test_resample_empty_series(freq, empty_series_dti, resample_method):
```

Comment:
IIUC the all_ts decorator is running this test for date_range, period_range and timedelta_range, not just dti